lib/circuitry/subscriber.rb

```diff
@@ -54,7 +51,7 @@ def subscribe(&block)
       logger.info("Unsubscribed from queue: #{queue}")
     rescue *CONNECTION_ERRORS => e
       logger.error("Connection error to queue: #{queue}: #{e}")
-      raise SubscribeError.new(e)
+      raise SubscribeError, e.to_s
```

Comment:
Do `e.to_s` and `e.message` return the same thing?  Might be safer to use the latter.